CONTRIBUTING.md

```diff
@@ -156,6 +167,11 @@ follow these steps to build `influxd` from source and start the service:
 
    `influxd` logs to stdout by default.
 
+**Troubleshooting**
+
+- If you've changed go or rust versions and are having trouble building, consider running `go clean -r -x -cache -testcache -modcache ./`
+- This will clear out any old build artifacts that may be incompatible.
```

Comment:
These don't really make sense as bullet points, they're part of the same thought.